src/vaults/vaults.utils.spec.ts

```diff
@@ -41,14 +45,14 @@ describe('vaults.utils', () => {
       }
       const startTime = opts.timestamp_gte;
       const data = [0, 1, 2].map((int) => {
-        const timestamp = Number((startTime + int * 20000).toFixed());
+        const timestamp = Number((startTime + int * ONE_DAY_SECONDS * 14).toFixed());
         const block = Number((timestamp / 10000).toFixed());
         return {
           timestamp,
           harvests: [{ timestamp, block, harvested: BigNumber.from((int + 1 * 1.88e18).toString()) }],
           treeDistributions: [
-            { timestamp, block, token: TOKENS.SUSHI, amount: BigNumber.from((int + 1 * 5.77e12).toString()) },
-            { timestamp, block, token: TOKENS.FARM, amount: BigNumber.from((int + 1 * 4.42e12).toString()) },
+            { timestamp, block, token: TOKENS.BCVXCRV, amount: BigNumber.from((int + 1 * 5.77e12).toString()) },
```

Comment:
these tokens were updated to make sure the "underlying" yield from these vaults gets added in tests